client/src/components/TaskDetails/TaskDetails.module.css

```diff
@@ -19,6 +19,17 @@
   border-radius: 30px;
   overflow: auto;
   padding: 20px 0px;
+  z-index: 10;
+}
+
+.captureClicks {
```

Comment:
I would rename this to `preventClick`. Better yet, is there a way to integrate this into the modal? (So then whenever you use the modal CSS class, it'll already have the CSS that belongs with capture clicks

Reply:
Renamed. I don't think we can combine it with the modal because it would prevent clicks that are supposed to land on the model itself. See the different `z-value` for `preventClick` and the modal itself.

Reply:
I see, that makes sense! Noted.